### DISCUSSION TITLE
Moving existing backup data to new container

### DISCUSSION THREAD

**Discussion by N0Br41nZ** [felddy/foundryvtt-docker#415] [Q&A]
Hi,

I have a previous installation of Foundry that I want to migrate from. I fetched the data folded with SSH and I was expecting to be able to dump it in the correct location using a SSH sync program. However, it seems that Docker is not letting me write in said folder using the normal SSH connection. Upon further research, it looks like Docker is jealous of its folders, permission-wise.

How can I access the data folder remotely, either through SSH or any other method, to sync my folders or simply copy over my worlds/modules/systems?

**Comment by felddy** [comment]
Could you provide some information about your container's configuration?  What are you using for storage?  Do you have a volume mounted for `/data` as in the documentation example?

**Reply by N0Br41nZ** [reply]
Yes, I have followed [this guide](https://www.youtube.com/watch?v=p9C8wfW6vC4&t=637s) to the letter. 
I specified "/home/<username>/foundryvtt" as my data folder when installing foundry. That's on the microSD of my raspberry.

I currently have 1 environment with 1 stack (foundry), 2 images (foundry, portainer), 2 containers (portainer, foundry), 4 network (bridge, foundry, host, none) and 1 volume (portainer_data. Mounted at: /data. Mount path: /var/lib/docker/volumes/portainer_data/_data. Container using volumes: portainer).

By writing all this down,I realized that perhaps there should be a volume for foundryvtt as well, though I assumed that the user data part of the installation could be simply added.

Edit: unless the portainer one is the one used by foundry. It matches the path.
